app-frontend/src/assets/styles/sass/_shame.scss

```diff
@@ -39,26 +39,57 @@ div.image-placeholder {
 }
 
 .editable-image {
+  $background: rgba(0,0,0,0.7);
```

Comment:
If we end up deleting this class from the html, I think we could remove this class css definition as well since it does not appear to be used in anywhere else.